server/test/org/apache/cloudstack/network/lb/CertServiceTest.java

```diff
@@ -726,17 +729,17 @@ public void runDeleteSslCertInvalidId() throws NoSuchFieldException, IllegalAcce
         when(certService._lbCertDao.listByCertId(anyLong())).thenReturn(null);
 
         //creating the command
-        DeleteSslCertCmd deleteCmd = new DeleteSslCertCmdExtn();
-        Class<?> _class = deleteCmd.getClass().getSuperclass();
+        final DeleteSslCertCmd deleteCmd = new DeleteSslCertCmdExtn();
+        final Class<?> klazz = deleteCmd.getClass().getSuperclass();
 
-        Field certField = _class.getDeclaredField("id");
+        final Field certField = klazz.getDeclaredField("id");
         certField.setAccessible(true);
         certField.set(deleteCmd, certId);
 
         try {
             certService.deleteSslCert(deleteCmd);
             fail("Delete with an invalid ID should fail");
-        } catch (Exception e) {
+        } catch (final Exception e) {
             assertTrue(e.getMessage().contains("Invalid certificate id"));
         }
```

Comment:
Please consider replacing this `try` block with `expected` on the `@Test` annotation